### DISCUSSION TITLE
Theming volume slider: GtkScale handle/knob thing

### DISCUSSION THREAD

**Discussion by ToxicMushroom** [JakeStanger/ironbar#768] [General]
![image](https://github.com/user-attachments/assets/1761f07f-9610-43f2-9ca9-b1196bba7226)
I've been struggling to find how to theme the white square that indicated where to grab and the position of the GtkScale
The https://docs.gtk.org/gtk3/class.Scale.html docs don't seem to mention it ?

**Comment by JakeStanger** [comment]
Hey, that can be targeted using the GTK `slider` element as the selector. Specifically, the full selector would be:

```css
.popup-volume .apps-box .app-box .slider slider
```

**Reply by ToxicMushroom** [reply]
Thanks it appears that it is a background-image.
Would you have any clue as to why it'd display as a square by default
I tried changing themes ect using the inspector but no luck
![image](https://github.com/user-attachments/assets/7cacf266-febe-4478-89d3-1c8bc8f6bf75)

If you've no idea I'll just theme it custom :)

**Reply by JakeStanger** [reply]
I'm not sure, it seems like an odd default. You can use `border-radius: 100%` to get a circle though.

**Comment by Diax170** [comment]
hi, how to you truncate the sink name? (in your case "Yeti Stereo Microphone")

**Reply by ToxicMushroom** [reply]
I don't know, I think this was the default.

**Reply by Diax170** [reply]
for me it displays full device name and I wonder how to truncate it

**Reply by Diax170** [reply]
could you share your `volume` config please?

**Reply by ToxicMushroom** [reply]
sure but do keep in mind that this is for 0.17 gtk3
```
[[end]]
type = "volume"
format = "{icon} {percentage}%"
max_volume = 100

[end.icons]
volume_high = "󰕾"
volume_medium = "󰖀"
volume_low = "󰕿"
muted = "󰝟"
```

**Reply by JakeStanger** [reply]
The difference here is likely caused by the [change in widget](https://github.com/JakeStanger/ironbar/wiki/gtk4#volume) between the GTK3 and 4 version. This behaviour is not currently configurable.
